app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+
+  def create
+    super
+  end
```

Comment:
This looks kinda better.

```suggestion
  before_action :configure_sign_up_params
```